enaml/qt/qt_scintilla.py

```diff
@@ -335,13 +437,30 @@ def pull_color(name, default):
             pull_bool('backspace_unindents', False))
         send(w.SCI_SETINDENTATIONGUIDES,
             pull_enum(INDENTATION_GUIDES, 'indentation_guides', 'none'))
+        w.setAutoIndent(pull_bool('auto_indent', False))
 
         # White Space
         send(w.SCI_SETVIEWWS, pull_enum(WHITE_SPACE, 'view_ws', 'invisible'))
         send(w.SCI_SETWHITESPACESIZE, pull_int('white_space_size', 1))
         send(w.SCI_SETEXTRAASCENT, pull_int('extra_ascent', 0))
         send(w.SCI_SETEXTRADESCENT, pull_int('extra_descent', 0))
 
+        # Autocompletion
```

Comment:
The widget documentation points the user the the SETTINGS file for documentation about the settings, but you not appear to have updated this file, could you do it ?